### ISSUE TITLE
ipc: Error: Second instance already running

### ISSUE THREAD

**Issue by SpaceShawn** [spaceconcordia/space-pcd#1]
`ipc: Error: Second instance already running.`

This error is caused by a process invoked by PCD still running after PCD has stopped. 

How to deal with this?

**Comment by harley1011** [comment]
Đđrrfřřřřrřeřr mm đ rydrřyou ét

**Comment by SpaceShawn** [comment]
Could you rephrase that? :p

_Shawn Bulger_

_ConSat-1, *_Ground Station*

 Space Concordia http://www.spaceconcordia.ca

*H1029-7, *1455 Rue De Maisonneuve West

_Montréal, Québec *_H3G 1M8*

(514) 848-2424 ext. 2738
